❌ MixedLanguage Implementation keeps failing

The MixedLanguage setup (VHDL + Verilog) keeps failing. I do not think this is a code problem - there were no relevant code changes there. I think this might be a regression issue with GHDL?!

GHDL fails to import the `neorv32_Fomu_MixedLanguage_ClkGen` module:
```
3. Executing GHDL.
./board_tops/neorv32_Fomu_BoardTop_MixedLanguage.vhd:87:3:warning: instance "clk_inst" of component "neorv32_fomu_mixedlanguage_clkgen" is not bound [-Wbinding]
  Clk_inst : neorv32_Fomu_MixedLanguage_ClkGen
```

Yosys then fails to handle that module:
```
ERROR: cell type 'neorv32_Fomu_MixedLanguage_ClkGen' is unsupported (instantiated as 'clk_inst')
```

--- a/setups/osflow/synthesis.mk
+++ b/setups/osflow/synthesis.mk
@@ -7,10 +7,16 @@ neorv32-obj08.cf: ${DEVICE_LIB}-obj08.cf ${NEORV32_SRC}
 work-obj08.cf: neorv32-obj08.cf ${DESIGN_SRC} ${BOARD_SRC}
 	ghdl -a $(GHDL_FLAGS) --work=work ${DESIGN_SRC} ${BOARD_SRC}
 
+ifeq ($(strip $(NEORV32_VERILOG_ALL)),)
+READ_VERILOG =
+else
+READ_VERILOG = read_verilog ${NEORV32_VERILOG_ALL};
+endif
+
 ${IMPL}.json: work-obj08.cf $(NEORV32_VERILOG_ALL)
 	$(YOSYS) $(YOSYSFLAGS) \
 	  -p \
 	  "$(GHDLSYNTH) $(GHDL_FLAGS) --no-formal $(TOP); \
-	  synth_${YOSYSSYNTH} \
+	  $(READ_VERILOG) synth_${YOSYSSYNTH} \
 	  -top $(TOP) $(YOSYSPIPE) \
-	  -json $@" $(NEORV32_VERILOG_ALL) 2>&1 | tee yosys-report.txt
+	  -json $@" 2>&1 | tee yosys-report.txt

